src/components/FutureWeather/FutureWeather.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import "./FutureWeather.scss"
+
+import SingleFutureWeather from './SingleFutureWeather'
+
+
+const FutureWeather = (props) => {
+  const {weatherInfo} = props
+
+  console.log('Future Weather Info:', weatherInfo);
+  // const newArr = weatherInfo.slice(0, 7)
+  // console.log(newArr);
+
+  // Create a Child Comp, WeatherCard. Pass Item as a prop, and sdisplay there
+  return (
+    <div className='FutureWeather'>
+        <p>FutureWeather</p>
+        <div className="weather-container" style={{display: "flex", justifyContent:"center"}}>
+      {
+        weatherInfo.length > 0 ?
+        weatherInfo.slice(1, 7).map((item) => {
```

Comment:
Good that you're slicing the data here 👍🏼 
As I said in my comment in `App.js`, you could be slicing your data up there in the parent, and only passing down the sub-array to FutureWeather